### PR TITLE
Fix the css!

### PR DESCRIPTION
**Pull request by m6jones**
Matching the designs!
image

### CODE REVIEW

File: demo/data/base/new-learning-path.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		"startDate":null,
 		"endDate":null,
 		"isActive":true,
-		"description":""
+		"description":"TEST"
```
Comment (lloydwaddell) [inline demo/data/base/new-learning-path.json:9]: nit: is this necessary? This data object should represent a new learning path, which would not have a description

**Review by lloydwaddell — COMMENTED** [review]
Single pane template visual issues

1. grey bar at the bottom
2. activities bar is being cut off

single-pane

**Comment by tenbradley** [comment]
> Single pane template visual issues
> 
> 1. grey bar at the bottom
> 2. activities bar is being cut off
> 
> single-pane

Is that a horizontal scroll bar?

**Comment by tenbradley** [comment]
Looks good in primary-secondary. Horizontal width is messed up in single pane.

**Comment by lloydwaddell** [comment]
yes, that is a horizontal scroll bar, my bad

File: demo/data/base/new-learning-path.json (hunk 1 above)
Comment (m6jones) [reply demo/data/base/new-learning-path.json:9]: Oh, I did this to check if it was working. The api here isn't exactly representing. But I can change it.

**Review by tenbradley — APPROVED** [review]

**Pull request merged**